src/main/resources/assets/patchouli/lang/de_DE.lang

```diff
@@ -0,0 +1,93 @@
+# Item Names
+item.patchouli:guide_book.name=Reiseführer
+
+# Testing
+item.patchouli:test_book_1.name=Testbuch 1
+item.patchouli:test_book_1.landing=Nicht so toll wie eine Einführung, es ist aber trotzdem präsent.
+
+item.patchouli:test_book_2.name=Testbuch 2
+item.patchouli:test_book_2.landing=Einführungstext! Es unterstützt auch $(2)Farben$() und $(4)solche $(bold)Sachen$()!
+
+item.patchouli:test_completion.name=Fortschritts-Testbuch
+item.patchouli:test_completion.landing=Das Buch ändert die Farben, wenn du Einträge freischaltest!
+
+item.patchouli:intro_book.name=Patchouli
+item.patchouli:intro_book.subtitle=Dokumentation für jedermann
+item.patchouli:intro_book.landing=Mit $(item)Patchouli$() machst du es dir einfach, fortschrittliche freischaltbare $(thing)Bücher$(), zu lesen für Modifikationen und Modifikationenpakete!
```

Comment:
The second comma is misplaced. Also, I really don't like how you translated "mod" and "mod pack", I don't think those should be translated at all.

Reply:
However, the word "mod" is translated correctly. "Mod" is an unofficially abbreviation in german.